openwisp_monitoring/device/migrations/0009_update_device_status_for_disabled_critical_checks.py

```diff
@@ -0,0 +1,26 @@
+from django.db import migrations
+
+from openwisp_monitoring.device.settings import get_critical_device_metrics
+
+
+def update_critical_device_metric_status(apps, schema_editor):
+    Check = apps.get_model('check', 'Check')
+    DeviceMonitoring = apps.get_model('device_monitoring', 'DeviceMonitoring')
+    critical_metrics_keys = [metric['key'] for metric in get_critical_device_metrics()]
```

Comment:
There's some duplication here, this logic could be encapsulated in a method, eg: `critical_keys = DeviceMonitoring._get_critical_metric_keys()`